UIElements/backgroundMenu.py

```diff
@@ -0,0 +1,141 @@
+import os
+from kivy.uix.gridlayout import GridLayout
+from UIElements.fileBrowser import FileBrowser
+from kivy.uix.popup import Popup
+from DataStructures.makesmithInitFuncs import MakesmithInitFuncs
+from UIElements.backgroundPickDlg import BackgroundPickDlg
+from kivy.core.image import Image as CoreImage
+from PIL import Image as PILImage
```

Comment:
I'm running into an import error with this guy, I'm going to install it but I just wanted to make sure we need it 100%